src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -1530,5 +1531,25 @@ public Location get(LootGenerateEvent event) {
 				}
 			}, EventValues.TIME_NOW);
 		}
+
+		// PlayerShearEntityEvent - 1.15+ event values
+		if (Skript.getMinecraftVersion().getMinor() >= 15) {
```

Comment:
```suggestion
		if (Skript.isRunningMinecraft(1, 15)) {
```
Something like this or consider a method exists